internal/signalfx-agent/Makefile

```diff
@@ -2,6 +2,15 @@
 
 NUM_CORES ?= $(shell getconf _NPROCESSORS_ONLN)
 
+# build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
+GOCMD?= go
+GOTEST=$(GOCMD) test
+GO_BUILD_TAGS=""
+GOTEST_OPT?= -race -v -timeout 180s --tags=$(GO_BUILD_TAGS)
+
+# COVER_PKGS is the list of packages to include in the coverage
+COVER_PKGS := $(shell $(GOCMD) list ./... | tr "\n" ",")
```

Comment:
I wonder why they used comma as separator unlike the go test command itself, anyway, it is their convention, nothing to do about it.

Reply:
I'm not super familiar with this, but it looks like separating by commas is [standard for the `-coverpkg` option](https://go.dev/doc/build-cover#building).